userspace/libsinsp/filterchecks.h

```diff
@@ -798,6 +798,7 @@ class sinsp_filter_check_container : public sinsp_filter_check
 		TYPE_CONTAINER_HEALTHCHECK,
 		TYPE_CONTAINER_LIVENESS_PROBE,
 		TYPE_CONTAINER_READINESS_PROBE,
+		TYPE_CONTAINER_IP_NAME,
```

Comment:
suggestion: `TYPE_CONTAINER_IP_ADDR` (or `IP_ADDRESS`, or just `IP`)?

Reply:
I would prefer `TYPE_CONTAINER_IP_ADDR` instead of `..._NAME`.